frontend/public/components/conditions.tsx

```diff
@@ -1,11 +1,11 @@
 import * as React from 'react';
-
+import * as _ from 'lodash';
 import { Timestamp } from './utils';
 import { CamelCaseWrap } from './utils/camel-case-wrap';
 import { K8sResourceCondition } from '../module/k8s';
 
-export const Conditions: React.SFC<ConditionsProps> = ({ conditions }) => {
-  const rows = (conditions || []).map((condition: K8sResourceCondition, i: number) => (
+export const Conditions: React.SFC<ConditionsProps> = ({ conditions = [] }) => {
+  const rows = conditions.map?.((condition: K8sResourceCondition, i: number) => (
```

Comment:
This will cause a runtime error if we get `null` as a value instead of `undefined`.

```suggestion
export const Conditions: React.SFC<ConditionsProps> = ({ conditions }) => {
  const rows = conditions?.map?.((condition: K8sResourceCondition, i: number) => (
```

Reply:
Fixed